### PR TITLE
sync token

### PR DESCRIPTION
**Pull request by chriku**

### CODE REVIEW

File: github/src/main/kotlin/gropius/sync/github/TokenManager.kt [hunk 1]
```diff
@@ -47,4 +72,16 @@ class TokenManager(
             "SYNC_GITHUB_USER_NO_TOKEN"
         )
     }
+
+    suspend fun advertiseIMSUser(user: IMSUser) {
```
Comment (nk-coding) [inline github/src/main/kotlin/gropius/sync/github/TokenManager.kt:76]: missing docs

File: github/src/main/kotlin/gropius/sync/github/TokenManager.kt [hunk 2]
```diff
@@ -13,16 +19,37 @@ import org.springframework.stereotype.Component
 @Component
 class TokenManager(
     @Qualifier("graphglueNeo4jOperations")
-    private val neoOperations: ReactiveNeo4jOperations
+    private val neoOperations: ReactiveNeo4jOperations,
+    private val gropiusGithubConfigurationProperties: GropiusGithubConfigurationProperties
```
Comment (nk-coding) [inline github/src/main/kotlin/gropius/sync/github/TokenManager.kt:23]: missing param docs

File: github/src/main/kotlin/gropius/Application.kt [hunk 3]
```diff
@@ -40,6 +43,15 @@ class SyncConfiguration(
...
 }
 
+/**
+ * Configuration properties for the GitHub API
+ *
+ * @param loginServiceBase Base url for login service
+ */
+@ConstructorBinding
+@ConfigurationProperties("gropius.sync.github")
+data class GropiusGithubConfigurationProperties(val loginServiceBase: URI, val apiSecret: String)
```
Comment (nk-coding) [inline github/src/main/kotlin/gropius/Application.kt:53]: missing docs

**Review by nk-coding — CHANGES_REQUESTED** [review]

File: github/src/main/kotlin/gropius/Application.kt (hunk 3 above)
Comment (chriku) [reply github/src/main/kotlin/gropius/Application.kt:54]: ecb94f0567d1f38e47c5a29ef6d8aa002ab5c15c

File: github/src/main/kotlin/gropius/sync/github/TokenManager.kt (hunk 2 above)
Comment (chriku) [reply github/src/main/kotlin/gropius/sync/github/TokenManager.kt:24]: 52cd681a18d4e540bb40b2f4f2338d09232712e0

File: github/src/main/kotlin/gropius/sync/github/TokenManager.kt (hunk 1 above)
Comment (chriku) [reply github/src/main/kotlin/gropius/sync/github/TokenManager.kt:81]: f1bc74c0c29a3144748e5e25ad7dc92a22daaa34

**Review by nk-coding — APPROVED** [review]

**Review by ModellbahnFreak — APPROVED** [review]

**Pull request merged**